tests/behat/features/capabilities/book/book-create.feature

```diff
@@ -14,3 +14,16 @@ Feature: Create Book page
 
     Then I should see the book I just created
     And it should not show author information
+
+  Scenario: User can create "book" node as book by default
+    Given I am logged in as an "administrator"
+
+    When I am on "node/add/book"
+    Then I should see "- Create a new book -" in the "#edit-book-bid" element
+
+  Scenario: User can create a topic as a book
+    Given I am logged in as an "administrator"
+    Given Book structure is enabled for topics
+
+    When I am on "node/add/topic"
+    Then I should see "- None -" in the "#edit-book-bid" element
```

Comment:
This test confused me for a little bit since I thought we were adding "topics as book" to Open Social. After some puzzling I understand that we're making sure that we don't break Drupal core functionality of allowing other content types with book structure enabled to be outside of a book. That's a good test to add! I'd like to make some tweaks to make that clearer :) 

```suggestion
  Scenario: Enabling core book functionality for other node types allows the content to still be placed outside of a book
    Given I am logged in as an "viewer"
    And book structure is enabled for topic

    When I am on "node/add/topic"
    
    Then I should see "- None -" in the Book select field
```

1. I've updated the scenario label and think it's clearer this way
2. I've used "And" to group the two "Given" together and changed `Book` to `book` as well as made `topic` singular. That allows us to make our test step slightly more generic in case anyone wants to test book structure for event for example.
3. I've kept `node/add/topic` in this test and will replace it in the other test where I'm introducing "on the topic creation form" :) 
4. I've added another select field assertion, which means for with the changes in my other suggestion we should now be able to inspect all kinds of select fields :D 

The implementation for 4 is as follows:
```php

  /**
   * Ensure a select field contains an option.
   *
   * @Then I should see :option in the :locator select field
   * @Then should see :option in the :locator select field
   */
  public function thenShouldSeeOptionInTheSelectField(string $option, string $locator) : void {
    $field = $this->getSession()->getPage()->findField($locator);

    if (null === $field) {
      throw new ElementNotFoundException($this->getSession()->getDriver(), 'form field', 'id|name|label|value', $locator);
    }

    $opt = $field->find('named', ['option' => $option]);

    if (NULL === $opt) {
      throw new ElementNotFoundException($this->getDriver(), 'select option', 'value|text', $option);
    }
  }
```